symantec_endpoint_security/plugin.spec.yaml

```
@@ -816,3 +816,38 @@ actions:
         description: "Agent matching the search"
         type: agent
         required: false
+  blacklist:
+    title: "Blacklist"
+    description: "Blacklist MD5 or SHA256 hashes"
+    input:
+      hashes:
+        title: "Hashes"
+        description: "Hashes (MD5 or SHA256) to add to the blacklist. Note: only one type of hash is allowed at a time"
+        type: "[]string"
+        required: true
+        example: '["9de5069c5afe602b2ea0a04b66beb2c0"]'
+      description:
+        title: "Description"
+        description: "Description for the blacklist"
+        type: string
+        required: true
+        example: "Hashes banned via InsightConnect"
```

This is what we're using in the other vendor plugins

```suggestion
        example: "Hash Blacklisted from InsightConnect"
```